mottak/pom.xml

```diff
@@ -115,16 +115,27 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>no.nav.melosys</groupId>
+            <artifactId>altinn-correspondence-agency-external-basic</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
         <dependency>
             <groupId>no.nav.security</groupId>
             <artifactId>token-validation-spring</artifactId>
             <version>${token-support.version}</version>
         </dependency>
         <dependency>
-            <groupId>no.nav.tjenestespesifikasjoner</groupId>
-            <artifactId>altinn-correspondence-agency-external-basic</artifactId>
-            <version>${tjenestespesifikasjoner.version}</version>
+            <groupId>jakarta.jws</groupId>
+            <artifactId>jakarta.jws-api</artifactId>
+            <version>3.0.0</version> <!-- TODO -->
```

Comment:
TODO fordi?

Reply:
Var usikker på hvorfor det var nødvendig. Var en TODO for min del bare egentlig. Har fjernet dependencien.